sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/SqlManagementClientBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Sql.Tests
+{
+    [RunFrequency(RunTestFrequency.Manually)]
+    [ClientTestFixture]
+    [NonParallelizable]
+    public abstract class SqlManagementClientBase : ManagementRecordedTestBase<SqlManagementTestEnvironment>
```

Comment:
Fixture should be called SqlManagementTestBase, or similar

Reply:
Fixture seems inheritable. I didn't have to put on each test client.

Reply:
Since this is being used as a base class for the tests, the name should indicate the purpose.  It is a base class for tests, not a base class for management clients, is what I mean.

Consider, if we add additional setup unrelated to the management client in the future, the name will no longer make sense.